astrodb_utils/fits.py

```diff
@@ -284,13 +295,18 @@ def make_skycoord(header):
             print("RA_TARG or RA is required")
             ra = None
             return None
+        else:
+            ra = float(header.get('RA'))
     else:
         ra = float(header.get('RA_TARG'))
-        if ra > 360:
-            print("RA_TARG does not appear to be in degrees")
-            print(f"RA_TARG: {ra}")
-            print("RA_TARG should be in degrees")
-            return None
+        
+    if ra > 360:
```

Comment:
There's a bigger issue with this function: you define and store `ra_targ` and `dec_targ`, but if they are not present then you go and use `ra` and `dec`. But you only make the skycoord object from ra/dec, not from ra_targ/dec_targ.

I think you should consolidate that extra variable- store everything in ra/dec, don't make an extra _targ variable that can be lost in the function.